avaframe/runScripts/runProbAnaCom1DFA.py

```diff
@@ -21,6 +21,16 @@
 import avaframe.in3Utils.fileHandlerUtils as fU
 
 
+# -------USER INPUT ------------
+# set for mu and relTh your reference values (from your local_com1DFACfg.ini file)
+# also other parameters can be used for filtering, here for example only null simulations are taken into account
+# for testRelTh and testMu
+# probability configurations
```

Comment:
remove?

Reply:
removed last line but added in first line this: probability configurations are used to filter simulations for creating probability maps: